docs/development/development.md

```diff
@@ -0,0 +1,47 @@
+# Development
+
+We will describe everything related to getting started with development on tfchain here.
+
+## Setup work environment
+
+See [setup](./setup.md)
+
+## Get the code
+
+```
```

Comment:
```bash

Reply:
It's unclear what is commented here..

Reply:
For every highlighted command it should be a highlighted bash command like the following:
![Screenshot from 2023-01-25 13-36-41](https://user-images.githubusercontent.com/66883096/214553894-c6b771b4-c724-4cb3-9340-e5e09c6701e8.png)